fastlane/spec/actions_specs/carthage_spec.rb

```diff
@@ -666,6 +686,53 @@
           end
         end
       end
+
+      context "when specify use_xcframeworks" do
+        context "when command is archive" do
+          let(:command) { 'archive' }
+
+          it "raises an exception" do
+            expect do
+              Fastlane::FastFile.new.parse("lane :test do
+                  carthage(command: '#{command}', use_xcframeworks: true)
+                end").runner.execute(:test)
+            end.to raise_error("Use XCFrameworks option is available only for 'build', 'bootstrap', and 'update' command.")
+          end
+        end
+
+        context "when command is update" do
+          let(:command) { 'update' }
+
+          it "adds the use_xcframeworks option" do
+            result = Fastlane::FastFile.new.parse("lane :test do
+                carthage(command: '#{command}', use_xcframeworks: true)
+              end").runner.execute(:test)
+            expect(result).to eq("carthage update --use-xcframeworks")
+          end
+        end
+
+        context "when command is build" do
+          let(:command) { 'build' }
+
+          it "adds the use_xcframeworks option" do
+            result = Fastlane::FastFile.new.parse("lane :test do
+                carthage(command: '#{command}', use_xcframeworks: true)
+              end").runner.execute(:test)
+            expect(result).to eq("carthage build --use-xcframeworks")
+          end
+        end
+
+        context "when command is bootstrap" do
+          let(:command) { 'bootstrap' }
+
+          it "adds the use_xcframeworks option" do
+            result = Fastlane::FastFile.new.parse("lane :test do
+                carthage(command: '#{command}', use_xcframeworks: true)
+              end").runner.execute(:test)
+            expect(result).to eq("carthage bootstrap --use-xcframeworks")
+          end
+        end
```

Comment:
```suggestion
        context "when valid command is used for --use-xcframeworks option" do
          it "adds the use_xcframeworks option" do
            ['update', 'build', 'bootstrap'].each do |command|
              result = Fastlane::FastFile.new.parse("lane :test do
                carthage(command: '#{command}', use_xcframeworks: true)
              end").runner.execute(:test)
              expect(result).to eq("carthage bootstrap --use-xcframeworks")
            end
          end
        end
```

I think you can combine those cases into one case. Can you please do that?

Reply:
@ainame 
Thanks for your review and suggestion! 
But, I think the expectation should be like below.  
```
expect(result).to eq("carthage bootstrap --use-xcframeworks")
```
Except for that, I will take your suggestion in PR 👍